webrender/src/render_backend.rs

```diff
@@ -401,8 +401,7 @@ impl RenderBackend {
                         notifier.as_mut()
                                 .unwrap()
                                 .as_mut()
-                                .unwrap()
-                                .pipeline_size_changed(pipeline_id, Some(new_size));
+                                .unwrap();
```

Comment:
As far as I know, the only use of storing the known pipeline sizes is to compare them in this code. Since we don't need to notify any other code about changes to pipeline sizes, that means we can get rid of all of the code that compares them, as well as any code that stores them for later use. Basically, any code that now does not do anything with the result of comparisons can be removed.